components/odh-notebook-controller/config/manager/manager.yaml

```diff
@@ -21,7 +21,7 @@ spec:
         runAsNonRoot: true
       containers:
         - name: manager
-          image: quay.io/opendatahub/odh-notebook-controller:latest
+          image: $(odh-notebook-controller-image)
           imagePullPolicy: Always
           command:
             - /manager
```

Comment:
on the line below this one, in downstream we have 

```
          args: ["--oauth-proxy-image", "registry.redhat.io/openshift4/ose-oauth-proxy@sha256:4bef31eb993feb6f1096b51b4876c65a6fb1f4401fee97fa4f4542b6b7c9bc46"]

```

is there a reason to use tag here and hash in downstream? why not use hash everywhere, and put the corresponding tag as comment for easier reference?

Reply:
Also in addition to this maybe we have to update it as well

Reply:
sure, but probably not right now, if it's touched, who knows what may break

Reply:
Opened a tracking issue: https://issues.redhat.com/browse/RHOAIENG-9935